pycbc/workflow/coincidence.py

```diff
@@ -280,7 +280,9 @@ def create_node(self, statmap_files, background_files, tags=None):
             tags = []
         node = super(PyCBCMultiifoAddStatmap, self).create_node(statmap_files,
                                                             tags=tags)
-        if 'injections' in (tags+self.tags):
+        # Enforce upper case
+        ctags = [t.upper for t in (tags+self.tags)]
```

Comment:
As it's failed anyway, I'll add a 'spaces around operator' comment
(needs to be t.upper()?)

Reply:
Yeah, that would be it .. Oops. Thanks